Nexus.Sdk.Shared/ErrorHandling/CustomErrorsException.cs

```diff
@@ -0,0 +1,23 @@
+﻿
+namespace Nexus.Sdk.Shared.ErrorHandling
+{
+    public class CustomErrorsException : Exception
```

Comment:
Can this rather extend `NexusApiException` as I see the `NexusResponseHandler` now returns this exception instead of the `NexusApiException` which might break current applications using it like QPay and QBS? This will allow clients already using it to still cast it to a `NexusApiException` if needed

Reply:
It now extends from NexusApiException